data/hai/hai missions.txt

```diff
@@ -1932,3 +1932,73 @@ mission "Station Exploration Abandon Ship"
+					decline
+				`	"Why would you want to go to <destination>? Isn't that an Unfettered planet?"`
+					to display
+						or
+							has "visited system: Wah Ki"
+							has "Wanderers: Translation Machine: offered"
+							has "First Contact: Unfettered: offered"
+			`	The Hai pauses for a moment, as if unsure of what to say. "My teachings are more... appreciated among our northern brethren."`
+			choice
+				`	"Teachings?"`
+				`	"Alright, I'll take you there."`
+					goto agree
+				`	"Unfettered space is too dangerous for me. You'll have to find someone else."`
+					goto dangerous
+			`	An unreadable expression appears on the Hai's face. "How do I explain this... You have your preachers and wise men among the humans, yes? I am somewhat similar." He stops for a moment, realizing something. "Ah, apologies for my rudeness - you probably do not know who I am. My name is Hareek. Now, will you take me to <destination>?"`
+			choice
+				`	"So you're going to preach to the Unfettered?"`
+				`	"I can take you."`
+					goto agree
+				`	"Unfettered space is too dangerous for me. You'll have to find someone else."`
+					goto dangerous
+			`	"That is an oversimplification, but it is not something I can explain adequately to a human." He turns away, looking into the fog. "There are portentous events occurring in the galaxy, and I must warn others of the turmoil I have forseen. The Unfettered crave turmoil, so naturally they are more inclined to believe me." He gives a wry smile.`
+			choice
+				`	"Alright, I can take you there."`
+					goto agree
+				`	"Sorry, but this whole thing sounds a little too strange for me."`
+					goto strange
+				`	"Unfettered space is too dangerous for me. You'll have to find someone else."`
+			label dangerous
+			`	He snorts. "Dangerous? I am approaching 300 years old, and yet I do not fear a trip to Unfettered space." He walks away, clearly disappointed.`
+				decline
+			label strange
+			`	He sighs. "That's alright. I would not expect you to understand." He walks away, disappearing into the mist.`
+				decline
+			label agree
+			`	"Excellent. There are not many captains nowadays, Hai or human, who take proper precautions when traveling through space. I'm glad to see that you have some sense, at least." He lowers his voice.`
+			`	"Since you seem to be a trustworthy captain, I will tell you of some things I have seen. Many years ago, I defended our space against the Korath raids. One day, we were hit by a massive fleet, and were caught unprepared. Every single ship in my patrol group was destroyed, except for ours - the only one with a Keystone. I learned two things in my time with the Korath: that the old legends are true, and that the stability we have enjoyed as a species is ending. Changes in the galaxy are inevitable, and those changes are arriving soon." He begins to move towards your ship. "But let us not speak of this now. I have business to attend to."`
```

Comment:
```suggestion
			`	"Since you seem to be a trustworthy captain, I will tell you of some things I have seen. Many years ago, I defended our space against the Korath raids. One day, we were caught unprepared and hit by a massive fleet. Every single ship in my patrol group was destroyed, except for ours - the only one with a Keystone. I learned two things in my time with the Korath: that the old legends are true, and that the stability we have enjoyed as a species is ending. Changes in the galaxy are inevitable, and those changes are arriving soon." He begins to move towards your ship. "But let us not speak of this now. I have business to attend to."`
```

I'd also recommend swapping this order for both more dramatic effect and a subtle difference in Korath actions. With this change the fleet hits them when they can be caught unprepared, as it is now they are unprepared when the fleet hits them anyway.